docs/writing_a_plugin/import_automation.md

```diff
@@ -33,16 +33,16 @@ The full sources of the plug-in built in this article are part of the *Import SD
+To start developing the new import automation plug-in, create a new .NET project using the project template <span class="nowrap">`PiWeb-Import-Sdk Plugin`</span>. Enter `SimpleGeneratorPlugin` as project name and select `Import automation` as Plugin type. If you are using this guide to start your own import automation plug-in, use another project name that better fits your import automation. Since the project name is by default also used as plug-in id, make sure the project name is reasonably unique.
 
 {: .note }
-If you are missing the <span class="nowrap">`PiWeb-Import-Sdk Plugin`</span> project template, have a look at [Project templates]({% link docs/setup/development_environment.md %}#project-templates) for project template installation instructions.
+If you do not have a project template named <span class="nowrap">`PiWeb-Import-Sdk Plugin`</span>, take a look at [Installing project templates]({% link docs/setup/development_environment.md %}#installing-project-templates).
 
 The newly created project should now look similar to this:
 
 ![Project structure](../../assets/images/writing_a_plugin/import_automation/project_structure.png "Project structure"){: .framed }
 
-Let us have a look at the files created by the project template: The `manifest.json` file is the manifest of the plug-in. We will deal with it in the next step. In addition to the manifest file, the project template has also created four source code files for us: `Plugin.cs`, `ImportAutomation.cs`, `ImportRunner.cs` and `AutomationConfiguration.cs`. Each of these files contains a class of the same name:
+Let us have a look at the files created by the project template: The `manifest.json` file is the manifest of the plug-in. We will edit it in the next step. In addition to the manifest file, the project template has also created four source code files for us: `Plugin.cs`, `ImportAutomation.cs`, `ImportRunner.cs` and `AutomationConfiguration.cs`. Each of these files contains a class of the same name:
 - `Plugin` is the entry point of the plug-in. It acts as a factory for the import automation provided by our plug-in. The project template already set this up to create instances of `ImportAutomation`, so we do not need to change its implementation.
 - `ImportAutomation` represents our new import automation. It acts as a factory to delegate its two responsibilities: Firstly, it creates instances of `ImportRunner` thus determining what the import loop will do. Secondly, it creates instances of `AutomationConfiguration` to determine what import plan settings will be available in the *PiWeb Auto Importer* UI when our import automation is used.
 - `ImportRunner` implements the actual import loop. We will implement it to periodically upload a new measurement to the *PiWeb backend* in step 4.

```

Comment:
Now I have seen multiple variants:
* *PiWeb backend*
* *PiWeb* backend
* PiWeb backend
We should use one variant consistently.

Reply:
Using *PiWeb* backend now consistently.